content/en/docs/2023.7/Reference/Concepts/fundamentals/blocks/block-properties/common-properties.md

```diff
@@ -60,17 +79,21 @@ None
 
 - [BlockTimeoutException][]
 - [InvalidBlockTimeoutException][]
+- [RunAsException][]
+- [PropertyNullException][]
```

Comment:
Should be in alphabetical order but that rule is broken in so make places so *shrug*